src/components/Textarea/Textarea.js

```diff
@@ -0,0 +1,38 @@
+/**
+ *
+ * Textarea
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  commonDefaultProps,
+  commonPropTypes,
+} from '../../commonPropTypes/input';
+import StyledTextArea from './StyledTextarea';
+
+function Textarea({ onChange, value, ...rest }) {
+  const handleInputChange = e => {
```

Comment:
You don't need this function here. The target will be sent by default. You can simply remove it